### PR TITLE
[SDPA-5923] Remove permissions_by_term module

### PR DESCRIPTION
**Pull request by vincent-gao**
### Jira
https://digital-engagement.atlassian.net/browse/SDPA-5923

### Change
remove permissions_by_term out of from tide_core.

### CODE REVIEW

File: tide_core.module [hunk 1]
```diff
@@ -412,13 +406,11 @@ function tide_core_node_grants(AccountInterface $account, $op) {
  */
 function tide_core_node_access(NodeInterface $node, $op, AccountInterface $account) {
```
Comment (anthony-malkoun) [inline tide_core.module:407]: @vincent-gao if this was here because of PBT, should the whole hook go?

**Review by anthony-malkoun — CHANGES_REQUESTED** [review]
Vincent, should the hooks be removed totally? Do they do anything besides work with permissions by term? What about sites that still have authenticated content?

File: tide_core.module (hunk 1 above)
Comment (vincent-gao) [reply tide_core.module:407]: hey @anthony-malkoun . this was for https://digital-engagement.atlassian.net/browse/SRM-24 , but because of PBT.

**Comment by vincent-gao** [comment]
> Vincent, should the hooks be removed totally? Do they do anything besides work with permissions by term? What about sites that still have authenticated content?

hey @anthony-malkoun , the hooks are for the dashboard.  the sites, like SSP and VICPOL, will keep permissions_by_term module.

File: tide_core.module (hunk 1 above)
Comment (anthony-malkoun) [reply tide_core.module:407]: So this hook is required for sites that don't have PBT? I'm just not sure what we're trying to achieve here.

**Review by anthony-malkoun — APPROVED** [review]
Thanks for explaining Vincent.

**Pull request merged**